src/main.rs

```diff
@@ -148,21 +148,33 @@ async fn schedule_background_jobs(
     sync_complete_rx: watch::Receiver<bool>,
 ) {
     let sched = JobScheduler::new().await.unwrap();
+    let mut jobs = vec![];
     if app_config.read_node {
-        if let Some(block_retention) = app_config.read_node_block_retention {
+        if let Some(block_retention) = app_config.pruning.block_retention {
             let schedule = "0 0 0 * * *"; // midnight UTC every day
-            let job = snapchain::background_jobs::job_block_pruning(
+            let job = snapchain::jobs::block_pruning::block_pruning_job(
                 schedule,
                 block_retention,
                 block_store.clone(),
                 shard_stores.clone(),
                 sync_complete_rx,
             )
             .unwrap();
-            sched.add(job).await.unwrap();
+            jobs.push(job);
         }
     }
-    // Other background jobs can be added here
+
+    let event_pruning_job = snapchain::jobs::event_pruning::event_pruning_job(
```

Comment:
Do we want this job running at the exact same time on all validators? I think it'd make sense to stagger because this is an expensive operation.

Reply:
Based on testnet, this seems to be fine. The first prune will be taxing, but will make sure that happens after backfill is complete. If it becomes a problem, we can move to pruning on commit